doc/website/getting-started/examples/icehello.md

```diff
@@ -0,0 +1,5 @@
+---
+title: How to send data to another process
```

Comment:
This is also somehow confusing. Maybe just
Start communicating via iceoryx or how to send data to another process.

Reply:
I think How to send data to another process is ok. Somehow does not sound "quite right" but is clear.
Alternatively Sending data to another process (via iceoryx, but this should be clear). Also, the overall style should be kept similar (not "How to XYZ" in some and this style in others if possible).